modules/ROOT/pages/_partials/commons/common-dbreplica.adoc

```diff
@@ -22,9 +22,9 @@ include::{root-partials}_show_page_header_block.adoc[]
 
 include::{root-partials}block-caveats.adoc[tag=enterprise-only]
 
-Couchbase Lite supports replication between two local databases.
+Couchbase Lite supports replication between two local databases at the database, scope, or collection level.
```

Comment:
"databases at the database" looks a bit odd? Hmm, probably fine, just looked weird viewing the edit out of context.